app/src/main/java/org/schabi/newpipe/player/Player.java

```diff
@@ -742,6 +745,20 @@ public void handleIntent(@NonNull final Intent intent) {
         NavigationHelper.sendPlayerStartedEvent(context);
     }
 
+    private void directlyOpenFullscreenIfNeeded() {
+        if (fragmentListener != null
+                && PlayerHelper.isStartMainPlayerFullscreen(service)
+                && DeviceUtils.isTablet(service)
+                && videoPlayerSelected()
+                && PlayerHelper.globalScreenOrientationLocked(service)) {
```

Comment:
Why do we only need the last condition? The player does not start in fullscreen mode when using the tablet emulator with auto-rotation and start main player in fullscreen enabled. That's irritating and unexpected behaviour for me.

Reply:
Because you can rotate the device and get fullscreen easily with unlocked autorotation. Totally expected like everywhere in the app

Reply:
Yes, this is the standard behaviour. I'll explain this is the case in the setting description.